sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.netstandard2.0.cs

```diff
@@ -29,14 +29,19 @@ public enum ServiceVersion
     public partial class TableServiceClient
     {
         protected TableServiceClient() { }
+        public TableServiceClient(System.Uri endpoint, Azure.Data.Tables.TableClientOptions options = null) { }
         public TableServiceClient(System.Uri endpoint, Azure.Data.Tables.TablesSharedKeyCredential credential, Azure.Data.Tables.TableClientOptions options = null) { }
```

Comment:
We should have one .ctor that takes no optional parameters, [per the guidelines.](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-client-constructor-no-default-params)